### PR TITLE
Frontend file skeleton created, no components

### PR DESCRIPTION
**Pull request by emilterrero**
resolving issue 'Create Frontend #3'

### CODE REVIEW

File: front-end/slothstagram/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (Underwaterr) [inline front-end/slothstagram/.gitignore:23]: Let's consolidate this `.gitignore` file into the root-level `.gitignore` so there is only one file!

File: front-end/slothstagram/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.1.1",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
```
Comment (Underwaterr) [inline front-end/slothstagram/package.json:12]: Are we using all of these dependencies? For instance, Jest. Let's add only what we need

**Review by Underwaterr — CHANGES_REQUESTED** [review]
Let's clean up some of the unused things, such as that React logo, so we start with a cleaner base

File: front-end/slothstagram/src/setupTests.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```
Comment (Underwaterr) [inline front-end/slothstagram/src/setupTests.js:5]: 👀

**Review by Underwaterr — CHANGES_REQUESTED** [review]

File: front-end/slothstagram/src/setupTests.js (hunk 3 above)
Comment (emilterrero) [reply front-end/slothstagram/src/setupTests.js]: Resubmitted again

**Review by Underwaterr — APPROVED** [review]
Yay!

**Pull request merged**